.eslintrc.cjs

```diff
@@ -163,11 +163,6 @@ module.exports = {
         browser: true,
         jquery: true,
       },
-      settings: {
-        react: {
-          pragma: 'h',
```

Comment:
What does this do?

Reply:
It's just no longer needed! See https://eslint-react.xyz/docs/changelog#v1230-2024-12-31 and https://eslint-react.xyz/docs/changelog#v150-2024-01-29.